rpc-test-client/test/simple-call.js

```diff
@@ -0,0 +1,36 @@
+
+let plasma = require('../index');
+let _ = require('lodash');
+let BigNumber = require('bignumber.js');
+
+
+it('should load client', (done) => {
```

Comment:
Can make this method `async`, and it'll catch errors automatically rather than having to use `done()`.